utils/configuration.go

```diff
@@ -109,6 +110,18 @@ func ParseSQLStorage(configuration *viper.Viper) (*Storage, error) {
 			"must provide a non-empty database source for %s",
 			store.Backend,
 		)
+	case store.Backend == notary.MySQLBackend:
+		// Parse the url into a  config object, and grab the string from it.
+
+		// cfg, err := sql.Open("mysql", store.Source)
+		url_config, err := mysql.ParseDSN(store.Source)
+		if err != nil {
+			return nil, err
+		}
+		if !url_config.ParseTime {
```

Comment:
Non-blocking:  We can probably just set `url_config.ParseTime` to true here, since if it was true before, setting it to true again doesn't hurt anything.  :)

Would be awesome if we could add a test for this though!